### ISSUE TITLE
[Bug] Tamed wolf cann't be beaten in the town and the wolf can inflict damage

### ISSUE THREAD

**Issue by AIphaDragon** [TownyAdvanced/Towny#2946]
### What steps will reproduce the problem?
Video: https://youtu.be/HECUdDQbp6Q

### What is your Towny version?
0.92.12

### What is your spigot/craftbukkit version?
Spigot 1.12.2

1. Full server startup from the logs\latest.log : https://pastebin.com/dPzEH1Zq
2. Towny config.yml: https://pastebin.com/Z2s5Mr7F

**Comment by LlmDl** [comment]
You have Animals in the "mob_types: Animals,NPC,Snowman,ArmorStand" line. Try using this list but without wolf: 'AbstractHorse, ChestedHorse, Chicken, Cow, Donkey, Horse, Llama, Mule, MushroomCow, Ocelot, Parrot, Pig, PolarBear, Rabbit, Sheep, SkeletonHorse, ZombieHorse,NPC,Snowman,ArmorStand'

**Comment by AIphaDragon** [comment]
Changed the configuration. Did not help.
Config: https://pastebin.com/tZ6m393P

**Comment by LlmDl** [comment]
I'll look into this.

So I don't forget, CombatUtil line 171.

**Comment by LlmDl** [comment]
Solved as of 0.92.0.13
```
- Solve tamed wolves being able to attack players who cannot defend themselves inside of plots with PVP off.
```
